views/term_table.erb

```diff
@@ -12,14 +12,14 @@
         <div class="container">
             <div class="term-navigation">
               <% if @prev_term %>
-                <a href="<%= term_table_url(@country, @house, @prev_term) %>" class="term-navigation__prev">
+                <a href="<%= term_table_url(@country, @house, @prev_term) %>" class="term-navigation__prev" data-ga-track-click="Selection toggle: bio">
                     <i class="fa fa-arrow-left"></i>
                     Earlier
                 </a>
               <% end %>
 
               <span class="term-navigation__jump">Jump to:
-                  <select class="js-navigation-menu">
+                  <select class="js-navigation-menu" data-ga-track-select="Selection toggle: bio">
```

Comment:
Should be `Jump to term by date` or something, rather than `Selection toggle: bio`